src/domains/Landing/OffersList/OffersList.styled.ts

```diff
@@ -0,0 +1,56 @@
+import { CardActions, CardMedia, Box } from '@material-ui/core';
+import { DefaultText, Header } from '@components/_universal/Typography.styled';
+import styled, { css } from 'styled-components';
+
+export const SliderContainer = styled(Box)(
+    ({ theme }) => css`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        width: 15rem;
+        margin-right: 2rem;
+    `,
+);
+
+export const SortingLabel = styled(DefaultText)(
+    ({ theme }) => css`
+        margin-right: 1rem;
+    `,
+);
+
+export const CardImage = styled(CardMedia)(
+    ({ theme }) => css`
+        height: 0;
+        padding-top: 56.25%;
```

Comment:
O tym magicznym paddingu słyszałem, nigdy tak nie robiłem, bo zawsze centrowałem flexem, ale jak działa to niech działa. Za to height: 0 ?